pkg/views/user/create/view.go

```diff
@@ -54,8 +56,17 @@ func CreateUserView(createView *CreateView) {
 				Validate(func(str string) error {
 					if str == "" {
 						return errors.New("password cannot be empty")
-					}
+					} else if len(str) < 8  {
```

Comment:
Thank you for your effort here.

We can just use regexp here, no need for a custom validator. Apply regexp to check the password and show a simple error message like 'Password is incorrect'. Also, let's add a note in the input field description about the expected password format.